corehq/ex-submodules/casexml/apps/phone/models.py

```diff
@@ -842,9 +842,6 @@ def save(self, *args, **kwargs):
         self.doc_type = "SyncLog"
         save_synclog_to_sql(self)
 
-    def delete(self):
-        delete_synclog(self._id)
```

Comment:
Sorry for the miscomms here. I meant to apply the same comment to `save` as well

Reply:
The `save` is little bit different on both the models, no?